views/homepage/steps.html

```diff
@@ -3,26 +3,47 @@
       <div class="content-wrapper">
 
         <div class="step">
-          <img class="example" src="img/goggles-step-1.png">
+          <figure class="example">
+            <img src="img/goggles-step-1.png">
+          </figure>
           <div class="description">
             <h2>Install Goggles</h2>
             <p>
               Make sure your bookmark bar is in view. Click and drag this
               button to your bookmark bar to install X-Ray Goggles.
             </p>
-            <a class="goggles-bookmark" href="javascript:"><span class="icon"></span>Drag and drop me in your bookmark bar</a>
+            <a class="goggles-bookmark" href="javascript:"><span class="dragme fa fa-arrows"></span>Drag and drop me in your bookmark bar</a>
```

Comment:
turns out this *needs* to be the phrase "X-Ray Goggles", or the bookmark will read "Drag and drop m...", which we don't want.